components/form/textInput/src/index.scss

```diff
@@ -0,0 +1,39 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+@mixin sui-text-input ($default-border: false, $default-focus: false) {
+  @include input-placeholder {
+    color: $c-form-text-input-placeholder;
+    opacity: 1;
+  }
+  color: $c-form-text-input-value;
+  font-size: $fz-form-text-input-inner-text;
+  font-weight: $fw-form-text-input-inner-text;
+  white-space: nowrap;
+
+  @if not($default-border) {
+    border: 0;
+
+    &:focus {
+      outline-width: 0;
+    }
+  }
+}
+
+.sui-FormTextInput {
+  &-value {
+    @include sui-text-input;
```

Comment:
i don't get the point of having a mixin here. 
does we pass `$default-border` or `$default-border` somewhere?

Reply:
Yes, there is not point as the default borders won't be used anymore.
I will change it in next commit.